packages/superset-ui-connection/test/callApi/parseResponse.test.ts

```diff
@@ -60,8 +60,7 @@ describe('parseResponse()', () => {
       .then(throwIfCalled)
       .catch(error => {
         expect(fetchMock.calls(mockTextUrl)).toHaveLength(1);
-        expect(error.stack).toBeDefined();
-        expect(error.message.includes('Unexpected token')).toBe(true);
+        expect(error.message.includes('Unexpected')).toBe(true);
```

Comment:
can you elaborate on what the new error is? I'm not necessarily against changing the tests just want to make sure the change isn't indicative of something else.

Reply:
The new error text is "Unexpected <" - slightly different, but probably not too important. The bigger possible concern is that `error.stack` is undefined. There is an assertion for it, do you think it's a dangerous change?

Reply:
hmm, interesting. I don't think the missing `error.stack` is a huge issue, it's a little weird though because if you `const e = new Error('test')`, `e.stack` is defined so maybe it's using some other `Error` type 🤔

Reply:
yeah looks like they have [their own `error` function that throws an object](https://github.com/sidorares/json-bigint/blob/master/lib/parse.js#L108) not an `Error`

Reply:
Okay, I've changed the code so that if JSONbig.parse fails, I catch the "error" and rewrap the message in a proper Error, so any code in `incubator-superset` should work as expected... thoughts?

Reply:
sounds good!